src/validation/validator.ts

```diff
@@ -0,0 +1,51 @@
+import { ValidationFactory } from './factory'
+import { FieldValidator } from './fieldValidators'
+import { Severity, Validation, ValidationResult } from './validation'
+
+export type FieldsValidators = {
+  [prop: string]: Array<FieldValidator>
+}
```

Comment:
this is not needed. inline this in the validate method

Reply:
uhm, I used it in the test too...